app/views/comments/thread.html.erb

```diff
@@ -78,7 +78,7 @@
   <% end %>
   <div role="list">
     <% comments.each do |comment| %>
-      <% if comment.deleted && !(current_user&.is_moderator && params[:show_deleted_comments] == "1") %>
+      <% if comment.deleted && !(current_user&.at_least_moderator? && params[:show_deleted_comments] == "1") %>
```

Comment:
Untested suggestion:

```suggestion
      <% if comment.deleted && (current_user&.standard? || params[:show_deleted_comments] != "1") %>
```

I really can't decide which is more readable, so this is just a suggestion for comparison.

Reply:
This one is a bit trickier, but, unfortunately, suffers from the same issue of negation + safe navigation + `at_least_moderator` not being equivalent to `&.standard?` ( due to `!(A & B)` -> `(!A | !B)` )